contents/handbook/people/hiring-process/engineering-hiring.md

```diff
@@ -44,39 +44,18 @@ The final stage of our interview process is the PostHog [SuperDay](/handbook/peo
 
 For full-stack roles, the task involves building a small web service (both backend and frontend) over a full day. The task is designed to be _too much_ work for one person to complete in a day, in order to get a sense of their ability to prioritize. 
 
+Each engineering SuperDay will have a SuperDay buddy, this person will conduct the interview halfway through the day, will be available in Slack throughout the day to answer any questions, they will also be giving feedback on the SuperDay output.
+
 An engineering SuperDay usually looks like this (_there is a degree of flexibility due to time zone differences)_:
 
 *   An invitation to a personal Slack channel for the SuperDay, which we'll use throughout the day
-*   Kick-off session with an engineer
+   * This will include:- 2x Ops team members, 2 cofounders, the interviewer from the technical screen & the SuperDay buddy      
```

Comment:
This was discussed on Slack a fair bit - (links to conversations below, for reference, if you haven't seen it)

Tim will basically take over the kick-off call Qs as he usually speaks to the candidate immediately afterwards so his call moves forward 30 mins and acts as a kick off call and an interview

I think it might also make sense, or would be a nice touch, for each SuperDay buddy to have a bit of a pre-prepared intro that they can just copy & paste into slack at the start of the day, trying to make them feel more comfortable. 

[Slack convo 1](https://posthog.slack.com/archives/C07PARVHGNR/p1730108102143109)
[Slack Convo 2](https://posthog.slack.com/archives/C07PARVHGNR/p1730495721353379)